lib/cli/provider/index.js

```diff
@@ -154,6 +159,9 @@ module.exports.handler = function(argv) {
   , filter: function(device) {
       return argv.serial.length === 0 || argv.serial.indexOf(device.id) !== -1
     }
+    , exclude: function(device) {
```

Comment:
Sounds good. I've read the code related to provider filter, the semantic of filter is that the provider  can only join the device whose serial name is contained in the argument of argv.serial, while the -exclude argument is added to exclude the devices match the Regexp. 

So, suggest to add the new option '-exclude' like what we see in the command of 'grep':

```
man grep:
     --include
             If specified, only files matching the given filename pattern are searched.  Note that
             --exclude patterns take priority over --include patterns.  Patterns are matched to the
             full path specified, not only to the filename component.
      
      --exclude
             If specified, it excludes files matching the given filename pattern from the search.
             Note that --exclude patterns take priority over --include patterns, and if no --include
             pattern is specified, all files are searched that are not excluded.  Patterns are
             matched to the full path specified, not only to the filename component.
```